my_scripts/run_script

```diff
@@ -1,17 +1,21 @@
 #!/bin/bash -l
 
-
-for num_nodes in 1 2 3 4 
+for num_pe in 16
```

Comment:
For this file as well, I would prefer you create a new file instead of modifying it

Reply:
Ok